directed_forgetting/experiment.js

```diff
@@ -66,6 +66,48 @@ var getInstructFeedback = function() {
 		'</p></div>'
 }
 
+var getTestFeedback = function() {
+	var global_trial = jsPsych.progress().current_trial_global
+	var data_length = 144
+	var start_cut = global_trial - data_length
+	
+	//below are counters to see if the subject is treating this task as a directed remembering as opposed to a directed forgetting task
+	var neg_respond_remember_total = 0
+	var pos_respond_remember_total = 0
+	var neg_respond_remember = 0
+	var pos_respond_remember = 0
+	
+	for(var b =0; b < data_length; b++){
+		if(jsPsych.data.getDataByTrialIndex(start_cut + b - 1).probe_type == 'neg'){
+		neg_respond_remember_total += 1
+			if(jsPsych.data.getDataByTrialIndex(start_cut + b - 1).key_press == '37'){
+			neg_respond_remember += 1
+			}
+		}else if (jsPsych.data.getDataByTrialIndex(start_cut + b - 1).probe_type == 'pos'){
+		pos_respond_remember_total += 1
+			if(jsPsych.data.getDataByTrialIndex(start_cut + b - 1).key_press == '39'){
+			pos_respond_remember_total += 1
+			}
+		}
+	}
+	
+	var directed_remembering_total = neg_respond_remember + pos_respond_remember
+	var directed_remembering_percent = directed_remembering_total / pos_respond_remember_total + neg_respond_remember_total
+
+	
+	if (directed_remembering_percent >= .75){
+	test_feedback_text = 'According to the pattern of your responses, we believe that you are treating this task as a directed remembering task.  Please remember that <strong>this is a directed forgetting task</strong>.</p>'+
```

Comment:
This instructions are a bit confusing. You say "when you are presented with a cue (i.e.TOP) you should..." There shouldn't be an "i.e." here. Just say "When you are presented with the TOP cue you should..."